datadog_sync/model/synthetics_global_variables.py

```diff
@@ -32,9 +32,19 @@ class SyntheticsGlobalVariables(BaseResource):
     # Additional SyntheticsGlobalVariables specific attributes
     destination_global_variables: Dict[str, Dict] = dict()
 
+    def pre_process(self, resource: Dict) -> None:
```

Comment:
Are you able to use the `pre_resource_action_hook` method instead to handle this since it runs before applying?

Reply:
Also, looks like `value` is only needed on create so it might be better to just handle it in the create step so we do not override values on subsequent updates.